docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/pagination/pagination.twig

```diff
@@ -25,14 +25,20 @@
     <ul class="civictheme-pager__items {{ pager_items_modifier_class }}">
       {# Print first item if we are not on the first page. #}
       {% if items.first is not empty %}
+        {% set text %}
+          {% include '@atoms/icon/icon.twig' with {
+            theme: theme,
+            symbol: 'arrows_leftarrow_2',
+          } %} {{ items.first.text|default('First') }}
+        {% endset %}
         <li class="civictheme-pager__item civictheme-pager__item--first hide-xxs show-l">
-          {% include '@molecules/directional-link/directional-link.twig' with {
+          {% include '@atoms/link/link.twig' with {
```

Comment:
`Link` has an icon support in it - any reason it was not used and a custom icon was used instead?